Traceback (most recent call last):
  File "C:\Program Files\Python312\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Program Files\Python312\Lib\site-packages\nbclient\client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python312\Lib\site-packages\jupyter_core\utils\__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python312\Lib\asyncio\base_events.py", line 691, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Program Files\Python312\Lib\site-packages\nbclient\client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "C:\Program Files\Python312\Lib\site-packages\nbclient\client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Program Files\Python312\Lib\site-packages\nbclient\client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import sys
sys.path.insert(0, r'D:\Github\molass-library')
from molass import get_version
assert get_version() >= '0.0.6'
------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mAssertionError[0m                            Traceback (most recent call last)
Cell [1;32mIn[1], line 4[0m
[0;32m      2[0m sys[38;5;241m.[39mpath[38;5;241m.[39minsert([38;5;241m0[39m, [38;5;124mr[39m[38;5;124m'[39m[38;5;124mD:[39m[38;5;124m\[39m[38;5;124mGithub[39m[38;5;124m\[39m[38;5;124mmolass-library[39m[38;5;124m'[39m)
[0;32m      3[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmolass[39;00m [38;5;28;01mimport[39;00m get_version
[1;32m----> 4[0m [38;5;28;01massert[39;00m get_version() [38;5;241m>[39m[38;5;241m=[39m [38;5;124m'[39m[38;5;124m0.0.6[39m[38;5;124m'[39m

[1;31mAssertionError[0m: 

